recipes/libwebp/all/conanfile.py

```diff
@@ -103,12 +103,20 @@ def package_info(self):
         self.cpp_info.components["webpdecoder"].set_property("pkg_config_name", "libwebpdecoder")
         self.cpp_info.components["webpdecoder"].libs = ["webpdecoder"]
         if self.settings.os in ["Linux", "FreeBSD"]:
-            self.cpp_info.components["webpdecoder"].system_libs = ["pthread"]
+            self.cpp_info.components["webpdecoder"].system_libs = ["m", "pthread"]
+
+        # sharpyuv
+        self.cpp_info.components["sharpyuv"].set_property("cmake_target_name", "WebP::sharpyuv")
+        self.cpp_info.components["sharpyuv"].set_property("pkg_config_name", "libsharpyuv")
+        self.cpp_info.components["sharpyuv"].libs = ["sharpyuv"]
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.components["sharpyuv"].system_libs = ["m", "pthread"]
```

Comment:
this component doesn't exist before 1.3.0

Reply:
ah yes thanks